src/components/career-form/career-form.tsx

```diff
@@ -158,9 +160,11 @@ export const CareerForm: React.FC<CareerFormProps> = (props) => {
           >
             <>
               {(!selectedFiles || selectedFiles.length === 0) && <Upload />}
-              <div>
-                Drop files to upload or <span>browse</span>
-              </div>
+              <Trans i18nKey="career.action.upload">
```

Comment:
1. Why not the `t()` function here (and on all other places where this component is used)?
2. I cannot find `career.action.upload` anywhere in the locales